src/webpack.js

```diff
@@ -74,11 +89,63 @@ const getBuildObjectFromBundle = callback => {
 };
 
 /**
- * @desc Watching webpack build process
+ * @desc Watching and Streamming webpack build process
  * @param {bool} devMode return true when arg is --dev
  * @param {function} callback return data
  */
-const watching = (devMode, callback) => {
+const watchAndStreamingData = (devMode, callback) => {
+  watchingChanges(devMode, () => {
+    getBuildObjectFromBundle(callback);
+  });
+};
+
+/**
+ * @desc Write bundle file
+ * @param {bool} devMode return true when arg is --dev
+ */
+const writeBundleFile = devMode => {
+  watchingChanges(
+    devMode,
+    () => {
+      writeBundleBuildFile();
+    },
+    true, //stop watching
+  );
+};
+
+/**
+ * @desc Write bundle build file
+ */
+const writeBundleBuildFile = () => {
+  const dAppMetaData = getDappMetaData();
+  const filePath = path.join(process.cwd(), 'dist/index.js');
+  const outputPath = path.join(process.cwd(), 'build/build.json');
```

Comment:
yoiu don't need to join the path. You can use a relativ path `./build/build.json`